### ISSUE TITLE
Dark theme not working with Social Fixer

### ISSUE THREAD

**Issue by MystiiG** [dtinth/dark-facebook#68]
I've used the Dark theme for quite some in on Facebook in Social Fixer.  This morning it no longer works.

**Comment by kirstenberwin** [comment]
Same here.  Using Safari (8.0.5) on a Mac (OS X 10.10.3).

**Comment by JoseHidalgo** [comment]
Same here. Using Firefox (latest version) on Windows Seven.

**Comment by fwd079** [comment]
Using Chrome on Windows 7 Professional. 
It started yesterday, I disabled the extension and restarted Chrome and enabled extension after Ctrl+Shift+Del and it worked. But today its broken again and yesterday's workaround isn't working anymore.

**Comment by fwd079** [comment]
Working today.

**Comment by pascalo7** [comment]
Same issue with the dark theme. Running windows 8.1 Chrome version 43.0.2357.65 m (X64)

**Comment by theonaciri** [comment]
Yup, Facebook might have changed something. Every SocialFixer's themes don't work properly anymore.

**Comment by pascalo7** [comment]
There is 2 or 3 that do work but they are to bright :D

**Comment by dtinth** [comment]
Have you tried the alternative URL and/or alternative methods described here?

https://github.com/dtinth/dark-facebook/wiki/Troubleshooting:-Theme-Stops-Working-Completely-on-Social-Fixer

**Comment by pascalo7** [comment]
yeah i'm using the dark Facebook css atm as userstyles was down last night

**Comment by theonaciri** [comment]
Thanks dtinth, thanks to the solution described in the wiki it works fine now !

**Comment by pascaldulieu** [comment]
ahh it must use user styles to grab the themes then?
